lib/gui/models/store.js

```diff
@@ -90,9 +89,17 @@ const storeReducer = (state, action) => {
 
       const newState = state.set('availableDrives', Immutable.fromJS(action.data));
 
-      if (action.data.length === 1) {
+      // Non-system drives
+      const noSystemDrives = _.filter(action.data, (drive) => {
+        return !drive.system;
+      });
+
+      const selectedDevice = newState.getIn([ 'selection', 'drive' ]);
 
-        const drive = _.first(action.data);
+      // There are non-system drives; set one as the selected drive.
+      if (!selectedDevice && noSystemDrives.length >= 1) {
+
+        const drive = _.first(noSystemDrives);
```

Comment:
Um, is this auto-selecting the first removable drive, if there are multiple drives available?
IMHO Etcher shouldn't do that, it should just stick with the current behaviour of only auto-selecting a drive if it's the **only** removable drive available.